nextflow_schema.json

```diff
@@ -68,6 +67,22 @@
             "fa_icon": "fas fa-dna",
             "description": "Reference genome related files and options required for the workflow.",
             "properties": {
+                "genome": {
```

Comment:
Do wen need this parameter?

Reply:
We don't use it but it would get reintroduced by the template anyway I think and we can ignore it.